packages/core/admin/ee/server/src/audit-logs/services/lifecycles.ts

```diff
@@ -79,9 +64,14 @@ const getRetentionDays = (strapi: Core.Strapi) => {
   return licenseRetentionDays;
 };
 
-const createAuditLogsService = (strapi: Core.Strapi) => {
+/**
+ * @description
+ * Manages the the lifecycle of audit logs. Accessible via strapi.get('audit-logs-lifecycles')
+ */
+const createAuditLogsLifecycle = (strapi: Core.Strapi) => {
```

Comment:
It's more clear and consistent with service in the name imo:
```suggestion
const createAuditLogsLifecycleService = (strapi: Core.Strapi) => {
```